### PR TITLE
volume list changed to volume get

### PR DESCRIPTION
**Pull request by mbasha-dell**
# Description
Documentation update for dellctl driver includes:

- Update for dellctl volume get feature

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/408|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Comment by JacobGros** [comment]
update help message please, it's updated to:
```
[root@master-1-EjaLwGdnYBDE0 bin]# ./dellctl volume get -h
lists stats on local volumes on cluster

Usage:
  dellctl volume get [flags]

Aliases:
  get, ls, list

Flags:
  -h, --help               help for get
      --insecure           provide flag to skip certificate validation
      --namespace string   namespace of the secret for the given tenant
      --proxy string       auth proxy endpoint to use

````

File: content/docs/references/cli/_index.md [hunk 1]
```diff
@@ -855,13 +858,12 @@ Lists PowerFlex volume infomation for a given tenant on a local cluster
...
+Gets PowerFlex volume infomation for a given tenant on a local cluster. The namespace is the namespace where tenant secret is created. 
 
 ```
-# dellctl volume list --proxy <proxy.dell.com> --namespace vxflexos
-NAME             VOLUME ID          SIZE       POOL     SYSTEM ID          PV NAME          PV STATUS   STORAGE CLASS   PVC NAME                NAMESPACE
-k8s-0325497cd7   a69b554f00000004   8.000000   mypool   636468e3638c840f   k8s-0325497cd7   Released    vxflexos        vol-create-test-cthdf   replication-suite-fe2eac41
-k8s-a0c031582b   a69b555000000005   8.000000   mypool   636468e3638c840f   k8s-a0c031582b   Released    vxflexos        vol-create-test-nqfwz   replication-suite-fe2eac41
-k8s-3908a6954f   a69b555100000006   8.000000   mypool   636468e3638c840f   k8s-3908a6954f   Released    vxflexos        vol-create-test-4flg5   replication-suite-fe2eac41
-k8s-28e4184f41   a69b554e00000003   8.000000   mypool   636468e3638c840f   none             none        none            none                    none
+# dellctl volume get --proxy <proxy.dell.com> --namespace vxflexos
```
Comment (JacobGros) [inline content/docs/references/cli/_index.md:864]: Output looks good, but can we improve output to show cases where PV (and/or) PVC is missing?

Comment (mbasha-dell) [reply content/docs/references/cli/_index.md:864]: @JacobGros Thanks for the review. Committed changes.

**Review by kumarkgosa — DISMISSED** [review]

File: content/docs/references/cli/_index.md [hunk 2]
```diff
@@ -840,28 +840,31 @@ dellemc/csi-vxflexos:v2.3.0     k8s1.24,k8s1.23,k8s1.22,ocp4.10,ocp4.9  k8s.gcr.
...
+NAME             VOLUME ID          SIZE       POOL     SYSTEM ID          PV NAME          PV STATUS   STORAGE CLASS   PVC NAME       NAMESPACE
+k8s-e7c8b39112   a69bf18e00000008   8.000000   mypool   636468e3638c840f   k8s-e7c8b39112   Released    vxflexos        demo-claim10   default
+k8s-e6e2b46103   a69bf18f00000009   8.000000   mypool   636468e3638c840f   k8s-e6e2b46103   Bound       vxflexos        demo-claim11   default
+k8s-b1abb817d3   a69bf19000000001   8.000000   mypool   636468e3638c840f   k8s-b1abb817d3   Bound       vxflexos        demo-claim13   default
+k8s-28e4184f41   a69b554e00000003   8.000000   mypool   636468e3638c840f   none             none        none            none           none
```
Comment (JacobGros) [inline content/docs/references/cli/_index.md:869]: This final row is not current output of cmd. Remove last row and add these rows:
``` NAME             VOLUME ID          SIZE       POOL     SYSTEM ID          PV NAME          PV STATUS   STORAGE CLASS   PVC NAME   NAMESPACE
k8s-28e4184f41   c6b2280d0000009a   8.000000   mypool   636468e3638c840f   k8s-28e4184f41   Available   local-storage  
k8s-7296621062   a69b554f00000004   8.000000   mypool   636468e3638c840f

```

**Review by JacobGros — CHANGES_REQUESTED** [review]

File: content/docs/references/cli/_index.md (hunk 2 above)
Comment (7koma95) [reply content/docs/references/cli/_index.md:869]: > This final row is not current output of cmd. Remove last row and add these rows:
> ``` NAME             VOLUME ID          SIZE       POOL     SYSTEM ID          PV NAME          PV STATUS   STORAGE CLASS   PVC NAME   NAMESPACE
> k8s-28e4184f41   c6b2280d0000009a   8.000000   mypool   636468e3638c840f   k8s-28e4184f41   Available   local-storage  
> k8s-7296621062   a69b554f00000004   8.000000   mypool   636468e3638c840f
> 
> ```

File: content/docs/references/cli/_index.md (hunk 1 above)
Comment (7koma95) [reply content/docs/references/cli/_index.md:864]: > Output looks good, but can we improve output to show cases where PV (and/or) PVC is missing?

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No grammatical edits are required.

**Review by JacobGros — APPROVED** [review]
LGTM

File: content/docs/references/cli/_index.md (hunk 1 above)
Comment (7koma95) [reply content/docs/references/cli/_index.md:864]: > Output looks good, but can we improve output to show cases where PV (and/or) PVC is missing?

Comment (7koma95) [reply content/docs/references/cli/_index.md:864]: > @JacobGros Thanks for the review. Committed changes.

**Review by kumarkgosa — APPROVED** [review]
looks good

**Review by bharathsreekanth — APPROVED** [review]

**Review by gallacher — APPROVED** [review]

**Review by mjsdell — APPROVED** [review]

File: content/docs/references/cli/_index.md (hunk 1 above)
Comment (7koma95) [reply content/docs/references/cli/_index.md:864]: > Output looks good, but can we improve output to show cases where PV (and/or) PVC is missing?

**Review by 7koma95 — APPROVED** [review]

**Review by 7koma95 — APPROVED** [review]

**Pull request merged**